### ISSUE TITLE
[Feature] Create localization framework

### ISSUE THREAD

**Issue by DeadlyFirex** [DeadlyFirex/PrimoTracker#14]
**Description**
The current reporting system in the microservice is not language aware (yet). 
This issue aims to address these issues and implement a service-wide localization system.

**Context**
The current response system is neither standardized, nor localized, this is causing several problems;

- Response texts are not localized.
- Response texts are not uniform, even if the same result.

**Expected Behavior**
- Implement a centralized localization system that handles translation.
- Integrate changes with #10.
- Structure log messages in a standardized format for easier parsing and analysis.
- Filter and aggregate logs to improve efficiency and reduce storage requirements.
- Store logs in a centralized location for easy retrieval and analysis.

**Use Case Examples**
- Users are able to specify language preferences in a request.
- Users can expect a consistently (localized) response. 

**Requirements**

- [ ] Research options for localization frameworks.
- [ ] Create localization files.
- [ ] Add language query parameters to existing endpoints.
- [ ] Integrate system into uniform response system.
- [ ] Testing and verification.
- [ ] Integration into deployment.

**Considerations**
- Ensure that the localization system is compatible with different environments.

**Additional Information**
N/A

**Dependencies**
- Depends-on: #10

**Related**
- Related-to: #7
- Related-to: #8

**Comment by DeadlyFirex** [comment]
Properly placed dependencies under their tab.

**Comment by DeadlyFirex** [comment]
After careful evaluation, this will introduce too much technical debt for a single version, and this is postponed until further evaluation, currently consistent responses are to be provided by the `response` framework.
Response framework: #10
